### ISSUE TITLE
Error when attempting to download a file

### ISSUE THREAD

**Issue by RowlandOti** [huxq17/Pump#9]
Using the wonderful library in app, but I consistently get the below error whenever I add something to be downloaded:

```java
 java.net.ProtocolException: Unexpected status line: ID3������`.COMM������������eng��https://musicrider.org/��TXXX������������Upload��Hunter��TIT2������������T��r��u��t��h�� ��H��u��r��t��s��TPE2������������L��i��z��z��o��TPE1������������L��i��z��z��o��TALB������%������U��.��K�� ��S��i��n��g��l��e�� ��C��h��a��r��t��s��TYER������������2��0��1��7��TCON������������(��1��3��)��P��o��p��WXXX��������������https://musicrider.org/APIC��/������image/jpg��37��������JFIF��,,��������C��
2019-10-19 13:08:12.183 12013-14652/com.kokonetworks.kokonet.client W/System.err:     at okhttp3.internal.http.StatusLine$Companion.parse(StatusLine.kt:78)
2019-10-19 13:08:12.185 12013-14652/com.kokonetworks.kokonet.client W/System.err:     at okhttp3.internal.http1.Http1ExchangeCodec.readResponseHeaders(Http1ExchangeCodec.kt:181)
2019-10-19 13:08:12.185 12013-14652/com.kokonetworks.kokonet.client W/System.err:     at okhttp3.internal.connection.Exchange.readResponseHeaders(Exchange.kt:105)
2019-10-19 13:08:12.185 12013-14652/com.kokonetworks.kokonet.client W/System.err:     at okhttp3.internal.http.CallServerInterceptor.intercept(CallServerInterceptor.kt:82)
2019-10-19 13:08:12.185 12013-14652/com.kokonetworks.kokonet.client W/System.err:     at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:112)
2019-10-19 13:08:12.186 12013-14652/com.kokonetworks.kokonet.client W/System.err:     at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.kt:37)
2019-10-19 13:08:12.186 12013-14652/com.kokonetworks.kokonet.client W/System.err:     at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:112)
```
I am using the below code for downloading:

```java
Pump.newRequest(fileRemotePath, fileDownloadPath.absolutePath)
            .threadNum(1)
            .forceReDownload(false)
            .setRetry(3, 200)
            .listener(object : DownloadListener() {

                override fun onProgress(progress: Int) {

                }

                override fun onSuccess() {
                    Toast.makeText(KOKONetApp.INSTANCE.applicationContext, "Download Finished", Toast.LENGTH_SHORT)
                        .show()
                }

                override fun onFailed() {
                    Toast.makeText(KOKONetApp.INSTANCE.applicationContext, "Download failed", Toast.LENGTH_SHORT)
                        .show()
                }
            })
            .submit()
```
what could be the issue?

**Comment by huxq17** [comment]
Could you please paste your remote url,or emai to huxq17@gmail.com.

**Comment by RowlandOti** [comment]
Sample urls are:

```java
http://41.80.133.249:8080/download?mediaUri=content://media/external/audio/media/11945
http://41.80.133.249:8080/download?mediaUri=content://media/external/audio/media/12161
http://41.80.133.249:8080/download?mediaUri=content://media/external/audio/media/12014
http://41.80.133.249:8080/download?mediaUri=content://media/external/audio/media/12137
```

**Comment by huxq17** [comment]
fix at 1.2.5.
